Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -633,7 +642,16 @@ private class PhotonActionSheetCell: UITableViewCell {
                 self.statusIcon.image = self.statusIcon.image?.createScaled(CGSize(width: 30, height: 30))
             }
             if statusIcon.superview == nil {
-                stackView.insertArrangedSubview(statusIcon, at: 0)
```

Comment:
we can just always call `statusIcon.removeFromSuperview()` to avoid having to have 2 nested if statements here.